src/terraboot/core.clj

```diff
@@ -13,6 +13,13 @@
 (def current-coreos-ami "ami-1807e377")
 (def ec2-ami "ami-bc5b48d0")
 
+(def all-external "0.0.0.0/0")
```

Comment:
I think any reference to an ami, account id, instance information and so on. all-external if it's a constant then I wouldn't be overly concerned about it, that can stay a string.  Anything that can change from client to client needs taking out IMHO.